spec/directions/composite_page_container/v1_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'spec_helper'
 
-RSpec.describe Kitchen::Directions::EocCompositePageContainer do
+RSpec.describe Kitchen::Directions::CompositePageContainer do
```

Comment:
since it looks like a bit of logic has been added to this class in order to accommodate EOB stuff, could a test be added for that here? I know it's already implicitly tested by `BakeReferences`, but I've found it's good to test things directly to catch things that might not be caught by the implicit case. I wouldn't be as worried about it if it was straightforward, but all of the cases in the template for `CompositePageContainer`are starting to get little hairy 💇